app/assets/javascripts/utils/api.js

```diff
@@ -600,6 +598,33 @@ const API = {
         label,
       };
     });
+  },
+
+  async checkArticleInWikiCategory(articles) {
+    const wiki_articles_title = Array.isArray(articles) ? articles : [articles];
```

Comment:
Ideally, the caller should always pass in an array. Would remove the need for this check